setup.py

```diff
@@ -153,7 +153,12 @@ def write_version_py(filename=_VERSION_FILENAME):
         install_requires = __requires__,
         license = "BSD",
         packages = find_packages(),
-        package_data = {'': ['images/*', '*.ini',]},
+        package_data = {'': ['images/*', '*.ini',],
```

Comment:
I don't think tox.ini would be included because it is not inside the envisage package.